packages/acceptance-tests/pkg-tests-specs/sources/commands/version.test.js

```diff
@@ -0,0 +1,21 @@
+describe(`Commands`, () => {
+  describe(`version`, () => {
```

Comment:
It's not a `version` command (I make the distinction because the v1 actually has a command called `version`, so it might be confusing to put this test here). Can you call the file `_entry.test.js` instead, and rename the group accordingly?

Reply:
Good point. Yah, I'll move this to `_entry.test.js`